app/configurator/interactive-filters/interactive-filters-config-state.tsx

```diff
@@ -16,26 +16,21 @@ export const useInteractiveFiltersToggle = ({
   path: InteractiveFilterType;
 }) => {
   const [state, dispatch] = useConfiguratorState(isDescribing);
-  const onChange = useCallback<(e: ChangeEvent<HTMLInputElement>) => void>(
-    (e) => {
-      const newIFConfig = produce(
-        state.chartConfig.interactiveFiltersConfig,
-        (draft) => {
-          if (draft?.[path]) {
-            draft[path].active = e.currentTarget.checked;
-          }
```

Comment:
I do not get why is this part (the .active = .. checked) removed ?

Reply:
It was removed because timeSlider do not use a "Show time slider" checkbox like all other interactive filters. I thought it was inconvenient to need to first toggle a checkbox and then select a dimension afterwards – but anyway we don't have design yet so this might change 😅

Reply:
Hmm, or maybe not, I will check :)